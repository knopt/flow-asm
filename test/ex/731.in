13
3
-2.342716148792956
0.8148574221122189 5.280981763675319 -8.544801357118493 1.5155851576880668 4.189680870823498 -2.7122757898830123 -7.349178828883254 9.841695978580425 4.954644342797225 4.259024112984163 -0.06093600514505937 -4.7292099489933985 2.6838265676683086 
2.7825472811422873 -9.72886749679996 2.5186635659618695 6.67767968457483 3.9314243657126458 3.357118429575536 -1.6327123397498884 -0.3837063786139865 5.677643861687635 -2.613447160205409 8.952823176886081 -4.727146100213027 1.3792721885004173 
-4.396440586602061 6.852055995816141 -7.447521856079 8.04310890752327 0.07787425400841386 6.384795996005678 -0.3869813627623646 -2.137532355647389 1.917347295309371 6.235622654482317 1.8365051529839622 -2.127723979395535 -7.509474488176789 

-2.492670562134272 9.335206076891932 5.568582974878966 
