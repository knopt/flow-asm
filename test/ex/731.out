-16.007040552073562 56.760782443567265 -64.0793138141453 10.640964913722353 5.230674767304947 -48.66477261075764 -56.68600029803192 100.95153289720514 4.318127350159363 15.406209406706699 -25.31824643016901 -47.72360566842076 40.468472885207575 
14.691044540142903 -150.24109169359278 63.85441796215753 94.06887294315226 1.947595824008291 14.869442171687831 -19.102440704489524 -0.978341583588713 38.93430448326465 -87.21248369621175 91.19937223804853 -69.1686817380531 55.9788230000852 
-76.72934340502565 81.58219160066199 -58.95077206722051 60.4742801578383 -43.071654435657905 34.00066587564811 -22.104302506540826 -11.529942878010406 7.998187445852267 38.390160758580805 -14.715916467288514 -31.28365616585051 -47.45940434158253 
